bot.ml

```diff
@@ -112,7 +108,7 @@ let headers header_list =
   Header.init ()
   |> (fun headers -> Header.add_list headers header_list)
   |> (fun headers -> Header.add headers "User-Agent" "coqbot")
-  |> fun headers -> Header.add_authorization headers credentials
+  |> (fun headers -> Header.add_list headers github_header)
```

Comment:
Instead of adding the github header here, you could do it every time `send_request` is used for GitHub (replace `[]` with `github_header`, just like for `gitlab_header`). That would be cleaner.